README.md

```diff
@@ -7,47 +7,50 @@
     <li><a href="#scala_library">scala_library/scala_macro_library</a></li>
     <li><a href="#scala_binary">scala_binary</a></li>
     <li><a href="#scala_test">scala_test</a></li>
+    <li><a href="#scala_repl">scala_repl</a></li>
+    <li><a href="#scala_library_suite">scala_library_suite</a></li>
+    <li><a href="#scala_test_suite">scala_test_suite</a></li>
+    <li><a href="#thrift_library">thrift_library</a></li>
     <li><a href="#scalapb_proto_library">scalapb_proto_library</a></li>
+    <li><a href="#scala_toolchain">scala_toolchain</a></li>
   </ul>
 </div>
 
 ## Overview
 
-This rule is used for building [Scala][scala] projects with Bazel. There are
-currently four rules, `scala_library`, `scala_macro_library`, `scala_binary`
-and `scala_test`.
+These rules are used for building [Scala][scala] projects with Bazel. There are
+currently rules for building binaries (and repl apps), libraries, tests, and
+protobuf and thrift libraries.
 
 ## Getting started
 
-In order to use `scala_library`, `scala_macro_library`, and `scala_binary`,
-you must have bazel 0.5.3 or later and add the following to your WORKSPACE file:
+In order to use these rules you must have bazel 0.5.3 or later and add the
+following to your WORKSPACE file:
 
 ```python
-
 rules_scala_version="a89d44f7ef67d93dedfc9888630f48d7723516f7" # update this as needed
 
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 http_archive(
-             name = "io_bazel_rules_scala",
-             url = "https://github.com/bazelbuild/rules_scala/archive/%s.zip"%rules_scala_version,
-             type = "zip",
-             strip_prefix= "rules_scala-%s" % rules_scala_version
-             )
+    name = "io_bazel_rules_scala",
+    strip_prefix = "rules_scala-%s" % rules_scala_version
+    type = "zip",
+    url = "https://github.com/bazelbuild/rules_scala/archive/%s.zip" % rules_scala_version,
+)
+
+load("@io_bazel_rules_scala//scala:toolchains.bzl", "scala_register_toolchains")
+scala_register_toolchains()
 
 load("@io_bazel_rules_scala//scala:scala.bzl", "scala_repositories")
 scala_repositories()
 ```
-To use a particular tag, use the tagged number in `tag = ` and omit the `commit` attribute.
-Note that these plugins are still evolving quickly, as is bazel, so you may need to select
-the version most appropriate for you.
+This will load the `rules_scala` repository at the commit sha
+`rules_scala_version` into your Bazel project and register a [Scala
+toolchain](#scala_toolchain) at the default Scala version (2.11.12)
 
-In addition, you **must** register `scala_toolchain` - To register default empty toolcahin simply add those lines to `WORKSPACE` file:
-```python
-
-load("@io_bazel_rules_scala//scala:toolchains.bzl", "scala_register_toolchains")
-scala_register_toolchains()
-```
-[read more here](#scala_toolchain)
+To use a particular tag, use the tagged number as the `rules_scala_version`
```

Comment:
I think we can also drop this as I think we don’t tag anymore

Reply:
yeah, there has not been a release in 3 years. fixed.